services/order.py

```diff
@@ -0,0 +1,32 @@
+from db.models import User

```

Comment:
Code Style: Use `get_user_model()` instead of `User`, it is the best practice. Replace the import of `User` with `get_user_model` from `django.contrib.auth` and use it to get the user.